packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -62,21 +84,23 @@ export default class PciTrainingJobsSubmitController {
       });
   }
 
-  onAddVolume(form) {
-    const volume = {
-      region: form.container.$viewValue.region,
-      container: form.container.$viewValue.name,
-      mountPath: form.mountPath.$viewValue,
-      permission: form.permission.$viewValue,
-    };
-
-    this.job.volumes.push(volume);
+  addVolume() {
+    this.job.volumes = [];
+    map(filter(this.httpHeader, { added: true }), (adder) =>
+      this.job.volumes.push({
+        region: adder.model.container.region,
+        container: adder.model.container.name,
+        mountPath: adder.model.mountPath,
+        permission: adder.model.permission,
+      }),
+    );
     this.filterContainers();
   }
 
   // Dirty hack to validate the oui-inline-adder form
   // eslint-disable-next-line class-methods-use-this
```

Comment:
Do we still need this comment ?
Regarding the `/submit/submit.html` file, you removed the `<oui-inline-adder>` and used the `<oui-field>`